src/main/kotlin/com/sourcegraph/cody/config/ui/AccountConfigurable.kt

```diff
@@ -120,9 +122,14 @@ class AccountConfigurable(val project: Project) :
 
     applyChannelConfiguration()
 
+    // todo: create som pub & sub mechanism for these
```

Comment:
important

Reply:
`s/som/some/g`

Reply:
While changing it could you add a bit more context there?
As I understand right now account type changes only if you reopen the dialog, but not if you change account type in the meantime, right?
Is it even realistic that we can change it? E.g. is there any external mechanism we could use for pulling such kind of info in an efficient way?